src/nendo_plugin_classify_core/plugin.py

```diff
@@ -171,7 +177,22 @@ def instruments(self, track: NendoTrack) -> dict:
         filtered_labels, _ = filter_predictions(
             predictions, settings.instrument_classes, threshold=0.05
         )
-        return {"instruments": filtered_labels[0]}
+        instruments = make_comma_separated_unique(filtered_labels)
+        return {"instruments": instruments}
+
+    @NendoAnalysisPlugin.plugin_data
+    def sfx(self, track: NendoTrack) -> dict:
+        """Compute the sfx of the given track."""
+        # SR=16000 as recommended in the essentia documentation
+        # https://essentia.upf.edu/reference/std_TensorflowPredictVGGish.html
+        signal = es.MonoLoader(filename=track.resource.src, sampleRate=16000)()
```

Comment:
```suggestion
        track_copy = track.copy()
        signal = track_copy.resample(16000)
        signal = signal[0] if signal.ndim == 2 else signal
```

Reply:
As advised @aaronabebe I got rid of the `MonoLoader` again and instead used the Nendo-native version of getting a version of the track that is sampled to 16khz